docs/src/main/asciidoc/continuous-testing.adoc

```diff
@@ -0,0 +1,146 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Quarkus - Continuous Testing
+
+include::./attributes.adoc[]
+
+:toc: macro
+:toclevels: 4
+:doctype: book
+:icons: font
+:docinfo1:
+
+:numbered:
+:sectnums:
+:sectnumlevels: 4
+
+Learn how to use continuous testing in your Quarkus Application.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven {maven-version}
+* The completed greeter application from the link:getting-started[Getting Started Guide]
+
+== Introduction
+
+Quarkus supports continuous testing, where tests run immediately after code changes have been saved. This allows you to
+get instant feedback on your code changes. Quarkus detects which tests cover which code, and uses this information to
+only run the relevant tests when code is changed.
+
+== Solution
+
+Start the link:getting-started[Getting Started] application (or any other application) using `mvn quarkus:dev`. Quarkus
+will start in development mode as normal, but down the bottom of the screen you should see the following:
+
+----
```

Comment:
I think you need proper source blocks to get this properly formatted on the website so you should add `[source]` before the block.